### ISSUE TITLE
Cannot find commented page

### ISSUE THREAD

**Issue by MaluNoPeleke** [mauricerenck/komments#19]
I have installed and integrated your plugin to my Kirby 3.6.0 alpha test installation and when I try to post a comment I get an error message the the page cannot be found:
![image](https://user-images.githubusercontent.com/1446541/129695552-fe490f40-c301-4e4f-9c36-d6344dfd911b.png)

> POST https://domain.tld/kirbytest/komments/send 404
> sendKomment @ komments.js:78
> (anonymous) @ komments.js:122

Could this be related to the 3.6 build?
Debugging turned on doesn't help.

**Comment by mauricerenck** [comment]
Unfortunately I didn't have the time yet to test the plugin with Kirby 3.6 so I cannot say if the beta is causing the error.

Can you please look into the network tab of your browser and post the JSON which is send to the `komments/send` endpoint when you submit the form?
This 404 likely appears when the target url of the comment cannot be translated to a kirby page and the json should contain that target url. This way we might track down the error.

**Comment by MaluNoPeleke** [comment]
```
{
  "komment": "test",
  "url": "",
  "email": "f.niedernolte@gmail.com",
  "author": "Frederik",
  "author_url": "https://www.peleke.de",
  "wmSource": "https://peleke7.uber.space/kirbytest/de/blog/mein-dritter-beitrag",
  "wmTarget": "https://peleke7.uber.space/kirbytest/de/blog/mein-dritter-beitrag",
  "wmProperty": "komment",
  "quote": "",
  "replyTo": "",
  "replyHandle": "",
  "cts": "10"
}
```

You can test it on the mentioned page.

**Comment by mauricerenck** [comment]
Okay, thank you.
Looks like the subfolder install causes this error, as I grab the path of the url and use it as a slug. I'll try to provide a bugfix for this.

**Comment by MaluNoPeleke** [comment]
Okay, great, looking forward to it.

**Comment by MaluNoPeleke** [comment]
Thanks that works :)

Only the blueprint doesn't:

> The field type "kommentspending" does not exist

```
fields:
      kommentsPending:
        type: kommentsPending
        label: Pending Komments
```

**Comment by mauricerenck** [comment]
> Edit: Clicking on "Answer" for an existing comment brings me to the home page, which shouldn't be the case I guess.

That's strange. Link link is just an anchor to the kommentform, no magic behind it. The only think happening, is setting some form values on click. May that be caused by something else? 

> btw: how to get the mentioned Dashboard? I cannot see it in my panel.
In the upper left, in the burger menu of kirby (where you can find the settings) there should be a komments entry

> Only the blueprint doesn't
I guess, I have to check that with Kirby 3.6 I don't have any issues with the current version. This will probably take some time.

**Comment by MaluNoPeleke** [comment]
You can see the behaviour on the mentioned page. The link points to https://domaint.tld/kirbytest/de#kommentform instead of https://domaint.tld/kirbytest/de/blog/mein-dritter-beitrag#kommentform

There is nothing showing up in the menu:
![image](https://user-images.githubusercontent.com/1446541/129892505-59ef78fe-8f60-4e52-b403-9c70a7c20ae8.png)

**Comment by mauricerenck** [comment]
> There is nothing showing up in the menu:

maybe that is a 3.6 thing, I have to test it

> You can see the behaviour on the mentioned page. The link points to https://domaint.tld/kirbytest/de#kommentform instead of https://domaint.tld/kirbytest/de/blog/mein-dritter-beitrag#kommentform

yes, but if you look at the source, you'll see:

![reply](https://user-images.githubusercontent.com/964547/129916578-e97c4e1b-4fc9-4d0b-b2d4-9251ba11bc7d.png)

So something must happen when the link is clicked. Maybe some Javascript doing stuff for page-transitions or something like that?

**Comment by MaluNoPeleke** [comment]
I am just using a modified Starterkit in a subfolder /kirbytest and I can already see in the source code / inspect mode by hovering over the mentioned #kommentform anchor link that it points to https://domaint.tld/kirbytest/de#kommentform but I don't know why as I don't have custom JS code that would modify it.
My .htaccess also has RewriteBase /kirbytest and I don't have this problem anywhere else so far.

**Comment by mauricerenck** [comment]
would be interesting to see, what happens, when you add an anchor link anywhere else on the page. Does it behave the same way?

**Comment by MaluNoPeleke** [comment]
Yes, you are right, sorry. I have to investigate why this is happening.
Summary: with the provided fix Komments can be used in 3.6.0 but without the dashboard and widget (only the blueprint for individual comments works).

**Comment by MaluNoPeleke** [comment]
Some additional information: I found out the reason for the anchor link behaviour: I am using the <base> element and anchor links need to have a leading slash like this to work: "/#komment": https://stackoverflow.com/questions/55031195/setting-base-path-breaks-anchor-links/55031437
Would you be able to add this @mauricerenck 
It shouldn't break the behaviour on pages without <base>
